scripts/regression/windows.ps1

```
@@ -0,0 +1,709 @@
+[CmdletBinding()]
+param(
+    [string]$ArtifactsRoot = (Join-Path $env:TEMP ("ntrace-regression-windows-" + (Get-Date -Format "yyyyMMdd-HHmmss"))),
+    [string]$TsharkPath = ""
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'TsharkPath' has been declared but not used.

(PSReviewUnusedParameter)